tests/sibilant/basics.lspy

```diff
@@ -35,6 +36,51 @@
 	  None))
 
 
+;; This can't be defined in the class, or it won't get called until runtime?
+(defmacro test-coro [macro-label def-expr . test-exprs]
```

Comment:
I am not a big fan of defining a global macro like this for use in one class, but I couldn't figure out how to keep it scoped to the class definition. Maybe I should just get over it and deal with the repetition…

Reply:
Rewrote this using `macro-let`.